### PR TITLE
the 'keep' param no longer seems to be used in CI job

### PR DESCRIPTION
**Pull request by pjfanning**

### CODE REVIEW

**Review by jrudolph — COMMENTED** [review]
lgtm

**Comment by pjfanning** [comment]
@jrudolph would you be able to try approving this PR again? The PR does not seem to have an approval and merging is blocked until there is one.

**Review by mdedetrich — APPROVED** [review]
lgtm

**Pull request merged**
